charts/osm/crds/meshconfig.yaml

```diff
@@ -187,6 +187,14 @@ spec:
                       description: Sets the service certificate validity duration, represented as a sequence of decimal numbers each with optional fraction and a unit suffix.
                       type: string
                       default: "24h"
+                experimental:
+                  description: Experimental configurations
```

Comment:
should we have another object multicluster under experimental and then the clusterId under that ? This way the entire multicluster object can be removed from the experimental section once its no longer a feature

Reply:
Done